code/binary-tree/src/test/java/ru/unn/agile/binaryTree/model/BinaryTreeCreateTests.java

```diff
@@ -0,0 +1,39 @@
+package ru.unn.agile.binarytree.model;
+
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class BinaryTreeCreateTests {
+    private BinaryTree binaryTree;
+
+    private int     simpleKey  = 42;
+
+    @Before
+    public void setUp() {
+        binaryTree = new BinaryTree();
+    }
+
+    @Test
+    public void canCreate() {
+        assertNotNull(binaryTree);
+    }
+
+    @Test
+    public void canAdd() {
+        String simpleData = "SimpleText";
+        binaryTree.add(simpleKey, simpleData);
+    }
```

Comment:
I don't see any assertions here, so the only one error you can get - compile time error. It's not enough for unit test as far as I see. So this test can be removed